source/confirm_conditions_met.html.md.erb

```diff
@@ -0,0 +1,31 @@
+---
+title: Confirm conditions met
+weight: 50
+---
+
+# Confirm conditions met
+Used to confirm that a candidate has met the conditions of an offer.
+
+```
+POST /applications/:uuid/confirm_conditions_met
```

Comment:
What do you think about about using hyphens for these? 

> use hyphens rather than underscores as word separators for multiword names

https://www.gov.uk/guidance/gds-api-technical-and-data-standards#when-naming-and-hosting-your-api

Reply:
I agree, same for `confirm_enrolment`